helm/cert-exporter/templates/cert-manager/cert-manager.yaml

```diff
@@ -0,0 +1,64 @@
+{{- include  "certManager.assertValidKind" . }}
```

Comment:
these files were moved from the "deployment" folder. I don't think this will have any negative impact, but are you aware of any?

If you could `git mv` them it would help with the review b/c there would be fewer diffs and it would preserve history.

Reply:
> I don't think this will have any negative impact, but are you aware of any?

None that I am aware of

Reply:
I have now split up the change and move+rename commits.

If you look at each commit separately it all now looks correct, but if you look at the commits together, it still looks like the files were deleted and new ons were added